.github/workflows/scripts/dependabot-pr.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -ex
+
+git config user.name $GITHUB_ACTOR
+git config user.email $GITHUB_ACTOR@users.noreply.github.com
+
+PR_NAME=dependabot-prs/`date +'%Y-%m-%d-%H%M%S'`
+git checkout -b $PR_NAME
+
+IFS=$'\n'
+requests=$(gh pr list --search "author:app/dependabot" --json number,title --template '{{range .}}{{tablerow .title}}{{end}}')
+message=""
+
+for line in $requests; do
+    if [[ $line != Bump* ]]; then 
+        continue
+    fi
+
+    module=$(echo $line | cut -f 2 -d " ")
+    if [[ $module == go.opentelemetry.io/collecto* ]]; then
```

Comment:
which packages are covered by `go.opentelemetry.io/collecto*` but not by `go.opentelemetry.io/collector*`?